### PR TITLE
Replace gsutil with variable path for pyinstaller

### PR DESCRIPTION
**Pull request by dan-garvey**

### CODE REVIEW

**Comment by jdluzen** [comment]
I just ran into this trying to get it working with embedded Python. Would also be great if `gsutil` was made optional if the files were already there.

File: web/index.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # from models.resnet50 import resnet_inf
 # from models.albert_maskfill import albert_maskfill_inf
-from models.stable_diffusion.main import stable_diff_inf
+from web.models.stable_diffusion.main import stable_diff_inf
```
Comment (powderluv) [inline web/index.py:3]: drop this change ?

File: shark/shark_downloader.py [hunk 2]
```diff
@@ -14,12 +14,24 @@
...
+
+def resource_path(relative_path):
+    """Get absolute path to resource, works for dev and for PyInstaller"""
+    base_path = getattr(
+        sys, "_MEIPASS", os.path.dirname(os.path.abspath(__file__))
+    )
+    return os.path.join(base_path, relative_path)
+
+
+GSUTIL_PATH = resource_path("gsutil")
```
Comment (powderluv) [inline shark/shark_downloader.py:32]: minor nit can we keep the GSUTIL_ variables together ?

**Review by powderluv — APPROVED** [review]

**Review by powderluv — APPROVED** [review]

**Comment by dan-garvey** [comment]
@jdluzen your issue should be resolved in #557

**Pull request merged**